Index: proyect/src/main/java/duoc/proyect/model/Curso.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package duoc.proyect.model;\r\n\r\npublic class Curso {\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/proyect/src/main/java/duoc/proyect/model/Curso.java b/proyect/src/main/java/duoc/proyect/model/Curso.java
--- a/proyect/src/main/java/duoc/proyect/model/Curso.java	(revision 1aeea950630f4e3e5c026fc319a691d11b3a0bd7)
+++ b/proyect/src/main/java/duoc/proyect/model/Curso.java	(date 1745868379908)
@@ -1,4 +1,5 @@
 package duoc.proyect.model;
 
 public class Curso {
+    
 }
